src/main/java/com/infoshareacademy/service/MenuService.java

```diff
@@ -1,14 +1,17 @@
 package com.infoshareacademy.service;
 
-import com.infoshareacademy.model.Answers;
-import com.infoshareacademy.model.Category;
 import com.infoshareacademy.model.Questions;
+import com.infoshareacademy.model.User;
 
+import java.util.List;
 import java.util.Scanner;
 
 public class MenuService {
 
+    private static User currentUser;
+
     public static void mainMenu() {
+        userMenu();
```

Comment:
Czemu tworzenie użytkownika zostało tu przeniesione? Każde wczytanie głównego menu np. po powrocie z menu kategorii każe teraz tworzyć nowego użytkownika.

Reply:
potrzebowałem informacji o graczu dla funkcjonalności gry, żeby móc przypisać gracza do rozgrywki.
Myślałem żeby to przekazywać jako argument funkcji mainMenu, ale musiałbym to powtarzać przy każdy wywołaniu tej funkcji w klasie MenuService. Pomyślałem, że tak będzie najlepiej.
Poprawiłem teraz żeby tworzenie gracza było wywołane tylko wtedy gdy go jeszcze nie ma.